app/controllers/pages_controller.rb

```diff
@@ -23,4 +23,20 @@ def tools
     @no_shared_search = true
     @no_shared_switch_service_link = true
   end
+
+  def update_cookies
+    remember_settings = params['cookie_remember_settings']
+    ga_tracking = params['cookie_consent_usage']
+
+    unless remember_settings.nil? || ga_tracking.nil?
+      set_cookie_policy(ga_tracking, remember_settings)
+    end
+  end
+
+  private
+
+  def set_cookie_policy(ga_tracking, remembers_settings)
+    policy = "{ \"settings\": \"true\", \"usage\": \"#{ga_tracking}\", \"remember_settings\": \"#{remembers_settings}\"}"
```

Comment:
Why not use similar to: https://github.com/trade-tariff/trade-tariff-frontend/pull/33/files#diff-e45c245cc345ee7e2ad5ebff6a1f94667e264108f1d911c8d29b098a4487de36R28 ?